package.json

```
@@ -32,9 +32,11 @@
     "@radix-ui/react-dropdown-menu": "^2.1.5",
     "@radix-ui/react-label": "^2.1.1",
     "@radix-ui/react-popover": "^1.1.5",
+    "@radix-ui/react-radio-group": "^1.2.2",
```

Biome (1.9.4)

[error] The key @radix-ui/react-radio-group was already declared.

This where a duplicated key was declared again.

If a key is defined multiple times, only the last definition takes effect. Previous definitions are ignored.

(lint/suspicious/noDuplicateObjectKeys)